lib/greet.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+
+module.exports = exports = {};
+var someName = process.argv[2];
```

Comment:
Great work storing the process.argv argument in a variable to use more easily later.